packages/components/src/Components/TagModal/TagModal.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './tagModal.scss';
+import { Modal } from '@patternfly/react-core';
+import classNames from 'classnames';
+import {
+    Table,
+    TableHeader,
+    TableBody
+  } from '@patternfly/react-table';
+
+
+export default class TagModal extends React.Component {
+    render() {
+        const { className, systemName, toggleModal, isOpen, rows, columns, ...props } = this.props;
+        return(
+            <Modal
+                {...props}
+                className={classNames('ins-c-tag-modal', className)}
+                isOpen={isOpen}
+                title={`Tags for ${systemName}`}
+                onClose={toggleModal}
+                isFooterLeftAligned
+            >
+                {this.props.children}
```

Comment:
Do you want to throw this up in the `const` above to keep it consistent with the other `this.props` stuff?